sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1058,89 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of the multinomial coefficient.
+        They can be expressed in numerous ways, including as a product of binomial coefficients or of factorials:
+    .. math:: {n \choose k_1, k_2, \ldots, k_m} = \frac{n!}{k_1!\, k_2! \cdots k_m!} = {k_1\choose k_1}{k_1+k_2\choose k_2}\cdots{k_1+k_2+\cdots+k_m\choose k_m}
+
+    The multinomial coefficients have a direct combinatorial interpretation,
+    as the number of ways of depositing n distinct objects into m distinct bins,
+    with k1 objects in the first bin, k2 objects in the second bin, and so on.
+
+    Examples
+    ========
+
+    >>> from sympy import Symbol, Rational, multinomial, expand_func
+    >>> n = Symbol('n', integer=True, positive=True)
+
+    >>> multinomial(15, 8)
+    6435
+
+    >>> multinomial(n, -1)
+    0
+
+    Rows of Pascal's triangle can be generated with the multinomial function:
+
+    >>> for N in range(8):
+    ...     print([multinomial(i, N-i) for i in range(N + 1)])
+    ...
+    [1]
+    [1, 1]
+    [1, 2, 1]
+    [1, 3, 3, 1]
+    [1, 4, 6, 4, 1]
+    [1, 5, 10, 10, 5, 1]
+    [1, 6, 15, 20, 15, 6, 1]
+    [1, 7, 21, 35, 35, 21, 7, 1]
```

Comment:
Can we remove the `for` loop and give some simple examples?

Reply:
This was meant for integers, right? IMO, a simple example for integers will do the work.

Reply:
examples like ?

Reply:
Ohhh I thought it was a cool way to demonstrate that multinomials can be used instead of binomials

Reply:
what kind of integer examples do you suggest?

Reply:
Any simple ones. https://en.wikipedia.org/wiki/Multinomial_theorem#Example

Reply:
Thank you ... that'll do